free-courses-es.md

```diff
@@ -104,6 +104,8 @@
 ### Frameworks
 
 * [Curso Django](https://www.pildorasinformaticas.es/course/django) - Juan Díaz (Píldoras Informáticas)
+* [Curso Django Udemy](https://www.udemy.com/course/probar-django-construir-una-aplicacion-web-en-python)- Justin Mitchel, Karlita K
+ (Udemy)
```

Comment:
Hello, small reason as to why the Travis CI build is failing. There needs to be a space after the ")" and the "-". I have showcased it below. 

```suggestion
* [Curso Django Udemy](https://www.udemy.com/course/probar-django-construir-una-aplicacion-web-en-python) - Justin Mitchel, Karlita K
 (Udemy)
```